docker/pg_tests/scripts/tests/restore_test.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/sh
+set -e -x
+
+PGDATA="/var/lib/postgresql/10/main"
+PGDATA_ALPHA="${PGDATA}_alpha"
+PGDATA_BETA="${PGDATA}_beta"
+ALPHA_PORT=5432
+BETA_PORT=5433
+
+# init config
+CONFIG_FILE="/tmp/configs/restore_test_config.json"
+COMMON_CONFIG="/tmp/configs/common_config.json"
+TMP_CONFIG="/tmp/configs/tmp_config.json"
+cp ${CONFIG_FILE} ${TMP_CONFIG}
+echo "," >> ${TMP_CONFIG}
+cat ${COMMON_CONFIG} >> ${TMP_CONFIG}
+/tmp/scripts/wrap_config_file.sh ${TMP_CONFIG}
+
+# init alpha cluster
+/usr/lib/postgresql/10/bin/initdb ${PGDATA_ALPHA}
+
+# preparation for replication
+cd ${PGDATA_ALPHA}
+echo "host  replication  repl              127.0.0.1/32  md5" >> pg_hba.conf
+echo "wal_level = replica" >> postgresql.conf
+echo "wal_keep_segments = 100" >> postgresql.conf
+echo "max_wal_senders = 2" >> postgresql.conf
+echo "hot_standby = on" >> postgresql.conf
+echo "listen_addresses = 'localhost'" >> postgresql.conf
+
+echo "archive_mode = on" >> postgresql.conf
+echo "archive_command = '\
+AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE \
```

Comment:
I guess there is no need to duplicate the settings before the each wal-g call. Instead, add them to the `restore_test_config.json` and pass the wal-g config using the `--config` flag (see the other tests for examples).

Reply:
Fixed.